doc/ReleaseNotes/ReleaseNotes.tex

```diff
@@ -228,35 +230,35 @@ \section{Distribution File}
+\section{Compiling MODFLOW~6}
+MODFLOW~6 has been compiled using Intel Visual Fortran and gfortran on the Windows and Mac/OS operating systems.  Because the program uses relatively new Fortran capabilities, newer versions of the compilers may be required for successful compilation.  For example, to use gfortran to compile MODFLOW~6, gfortran version 4.9 or newer must be used.  If you have gfortran installed on your computer, you can tell which version it is by entering ``\verb|gfortran --version|'' at a terminal window.
 
-This distribution contains the Microsoft Visual Studio project files for compiling MODFLOW 6 on Windows using the Intel Fortran Compiler.  The files have been used successfully with Visual Studio 2017 and Intel(R) Visual Fortran Compiler 2019.2.190.
+This distribution contains the Microsoft Visual Studio project files for compiling MODFLOW~6 on Windows using the Intel Fortran Compiler.  The files have been used successfully with Visual Studio 2017 and Intel(R) Visual Fortran Compiler 2019.2.190.
 
-This distribution also comes with a makefile for compiling MODFLOW 6 with gfortran.  The makefile is contained in the \texttt{make} folder.
+This distribution also comes with a makefile for compiling MODFLOW~6 with gfortran.  The makefile is contained in the \texttt{make} folder.
 
-For those familiar with Python, the pymake package can also be used to compile MODFLOW 6.  Additional information on the Python pymake utility can be found at: \url{https://github.com/modflowpy/pymake}.  
+For those familiar with Python, the pymake package can also be used to compile MODFLOW~6.  Additional information on the Python pymake utility can be found at: \url{https://github.com/modflowpy/pymake}.  
 
 % -------------------------------------------------
 \section{System Requirements}
-MODFLOW 6 is written in Fortran.  It uses features from the 95, 2003, and 2008 language.  The code has been used on UNIX-based computers and personal computers running various forms of the Microsoft Windows operating system.
+MODFLOW~6 is written in Fortran.  It uses features from the 95, 2003, and 2008 language.  The code has been used on UNIX-based computers and personal computers running various forms of the Microsoft Windows operating system.
 
 % -------------------------------------------------
 \section{Testing}
-The examples distributed with MODFLOW 6 can be run by navigating to the examples folder and executing the "run.bat" batch files within each example folder.  Alternatively, there is a runall.bat batch file under the examples folder that will run all of the test problems.
+The examples distributed with MODFLOW~6 can be run by navigating to the examples folder and executing the "run.bat" batch files within each example folder.  Alternatively, there is a runall.bat batch file under the examples folder that will run all of the test problems.
```

Comment:
is run.bat surrounded by non-ascii double quotes?

Reply:
Yes it is. It is something that has been in there awhile (maybe forever). Fixed it.